packages/js/client/README.md

```diff
@@ -70,15 +82,14 @@ const config = {
       },
     },
   ],
-  
-  // ADVANCED USAGE:
-  
   // customize URI resolution
-  resolver: new RecursiveResolver(
-    new PackageToWrapperCacheResolver(wrapperCache, [
-      new ExtendableUriResolver(),
-    ])
-  ),
+  resolvers: [
+    new RecursiveResolver(
+      new PackageToWrapperCacheResolver(wrapperCache, [
+        new ExtendableUriResolver(),
```

Comment:
Do we have any docs that explain the various resolvers, at least the ones that come with Polywrap by default?

This might help developers set up their own resolution logic.

I'm thinking we could add this to `packages/js/uri-resolver-extensions/README.md` and reference it here? @nerfZael thoughts?

Reply:
That's a good idea. I can work on it soon. I was also thinking to write more detailed reference documentation (in the readme) for the client config builder since it's becoming so important.

It would be cool to create good reference docs (as readmes) for wasm-js and plugin-js, and really every package.

Reply:
Ideally we'd have the package creators also write up documentation, or at least a basic outline of it (with functional details), and have someone else (e.g. me, you, any volunteer) fix it up to spec.

Reply:
yes, 100%